### DISCUSSION TITLE
blog/sqybi-com-relaunching-after-a-decade/

### DISCUSSION THREAD

**Discussion by giscus** [sqybi/SQYBI.com#1] [Announcements]
# blog/sqybi-com-relaunching-after-a-decade/

这十年以来，SQYBI.com 的首页一直是下面这段文字：

https://sqybi.com/blog/sqybi-com-relaunching-after-a-decade/

**Comment by sqybi** [comment]
还没有测试过这个评论区功能，自己发一条测试一下~

**Comment by zerolfx** [comment]
《文学少女》好！

**Comment by IceDream61** [comment]
看到你重新开始写东西真开心！学长加油~！

**Comment by jhwei** [comment]
cong！冒个泡

**Comment by xuejc1988** [comment]
可能是第一个RSS订阅者！

**Comment by abc881858** [comment]
牛的，我可是一路从百度空间追到校内最后到sqybi.com的
这是我的小站 https://thinkinginqt.com

**Reply by sqybi** [reply]
确实哈哈哈，回去就加链接！
其实我也一直在看到你这边写的东西，看到你一直在钻研 Qt，牛的牛的！

**Comment by Oreoxmt** [comment]
新年快乐 🥳

**Reply by sqybi** [reply]
新年快乐！

**Comment by Reedo0910** [comment]
写得很棒，我也经历过这段历史，那时看到装着FrontPage和Dreamweaver的电脑还特别羡慕。我那时也经常上百度空间写博客，直到它关站。看完这篇文章之后仿佛自己也重温了一遍那段岁月（笑）
其他文章也写得很用心啊，已经订阅RSS了！
（不知有没有兴趣加友链：https://www.velasx.com ）

**Reply by sqybi** [reply]
当然没问题！不如说已经加上了 :D
其实我已经偷偷订阅你的 RSS 好久了哈哈哈，甚至已经忘了是怎么看到的……我也很喜欢你的文字！

**Reply by Reedo0910** [reply]
感谢感谢www 已经加上了
